### PR TITLE
Add support for integrated kernel driver

### PR DESCRIPTION
**Pull request by velomatt**
If an env. var "INTEGRATED_DRIVER" is set to `true` (or similar) then modify the behavior of dss_host.py:

1. Never unload nvme or nvme-core drivers
2. Use modprobe instead of insmod for nvme-fabrics, nvme-tcp, and nvme-rdma
3. Do not modprobe nvme unnecessarilly (in `remove` function)

Additionally, I implemented a fix where we are indiscriminately attempting to process all nvme subsystems.
Now we are only processing RDMA subsystems.

### CODE REVIEW

File: host/scripts/dss_host.py [hunk 1]
```diff
@@ -445,13 +454,14 @@ def subnet_drive_map():
         for dev in subsys["Paths"]:
             name = '/dev/' + dev["Name"] + 'n1'
             transport = dev["Transport"]
-            addr = re.search(r"traddr=(\S+)", dev["Address"]).group(1)
-            subnet = getSubnet(addr)
+            if transport == "rdma":
```
Comment (grandsuri) [inline host/scripts/dss_host.py:457]: What about non-rdma path? Do we need to handle?

**Review by grandsuri — COMMENTED** [review]
Rest looks good.

File: host/scripts/dss_host.py (hunk 1 above)
Comment (velomatt) [reply host/scripts/dss_host.py:457]: I asked @hroogi for his feedback about that this morning. I'll let him elaborate if necessary, however currently TCP subsystem is not working, so in his opinion it's ok to explicitly only handle RDMA subsystems (for now).

**Review by grandsuri — APPROVED** [review]

**Comment by velomatt** [comment]
Build is presently failing due to an issue with Sonar-Scanner. https://community.sonarsource.com/t/403-errors-on-sonar-scanner-cli-since-v6-upgrade/116407/9

I'll re-run the build once there's a fix upstream.

**Comment by nsarras** [comment]
looks good

**Pull request merged**